jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/SessionManager.java

```diff
@@ -220,10 +217,10 @@ protected void save(boolean force) {
             delay *= 2;
           }
           logger.log(
-              Level.SEVERE, String.format("Session %s not saved: too many attempts", getId()));
+            Level.SEVERE, String.format("Session %s not saved: too many attempts", getId()));
         } catch (DeadlineExceededException e) {
           logger.log(
-              Level.SEVERE, String.format("Session %s not saved: too many timeouts", getId()), e);
+            Level.SEVERE, String.format("Session %s not saved: too many timeouts", getId()), e);
```

Comment:
the same. Was there anything changed in this file other than formatting?
If not, I would revert it as a side of the removal of the empty `@param` at the bottom
of the file, most seems to violate the guideline.